packages/app/src/app/pages/common/Modals/TemplateModal/index.js

```diff
@@ -2,67 +2,68 @@ import React, { useState } from 'react';
 import OutsideClickHandler from 'react-outside-click-handler';
 
 import { SketchPicker } from 'react-color';
-import { inject } from 'mobx-react';
 import { Button } from '@codesandbox/common/lib/components/Button';
 import Input, { TextArea } from '@codesandbox/common/lib/components/Input';
 import { Checkbox } from '@codesandbox/common/lib/components/Checkbox';
 import * as templates from '@codesandbox/common/lib/templates';
-
 import uniq from 'lodash-es/uniq';
+import { useStore, useSignals } from 'app/store';
 
 import { Heading, Container, Explanation } from '../elements';
 import {
   Fieldset,
   Label,
   DefaultColor,
-  GlobalStylesStarterModal,
+  GlobalStylesTemplateModal,
 } from './elements';
 
-const StarterModal = ({ store, signals }) => {
+const TemplateModal = () => {
+  const { editor, workspace } = useStore();
```

Comment:
@SaraVieira Why not destructuring `editor` even further? 🤔

Reply:
I commented, we discussed this between me and ives and we find it more confuysing